lib/request/complement.js

```diff
@@ -211,3 +192,5 @@ function findPartialIntersections(requestedPath, optimizedPath, currentTree, dep
     return [intersections, oComplementPaths, rComplementPaths];
 }
 
+// Exported for unit testing.
+module.exports.findPartialIntersections = findPartialIntersections;
```

Comment:
You could lose the comment above and prevent accidental usage of this function by doing:

```js
module.exports.__test = {findPartialIntersections: findPartialIntersections};
```